styles/output/american-government-pdf.css

```diff
@@ -1469,6 +1486,15 @@ span[data-media~=screenreader] {
   font-weight: 600;
 }
 
+[data-type=page].introduction .intro-text > section > section > section > h4 {
```

Comment:
Should this font match the following block? (I'm just going with the pattern I've seen so far)